### PR TITLE
Patient Roster spec updates for UNC

### PR DESCRIPTION
**Pull request by santry**

### CODE REVIEW

File: patient-roster.md [hunk 1]
```diff
@@ -72,8 +80,11 @@ More on each:
 - The reporting period duration is either `month` or `week`.
 - The reporting period start date is the first date of the reporting period in ISO 8601 basic date format. For example, `20180301` is the first of March, 2018.
 - The timestamp is a UTC date and time in ISO 8601 basic date/time format. For example, `20180314T190400Z` is March 14, 2018 at 19:04 (i.e., 7:04PM) in UTC, or the same date at 3:04PM in US Eastern time.
+- The patient roster specification version is the version number of this document, preceded by `v`. For example, `v1` indicates that the file conforms to version 1 of the patient roster specification
```
Comment (97jaz) [inline patient-roster.md]: period at the end of sentence

File: patient-roster.md [hunk 2]
```diff
@@ -102,6 +113,22 @@ Each record in a patient roster file contains the following fields, in the follo
 13. City
 14. State
 15. [ZIP code](#zip-format)
+16. Primary Care Physician name
+17. Primary Care Physician NPI
+18. Primary Care Physician [phone number](#phone-format)
+19. Primary Care Physician email
+20. Primary payer
```
Comment (97jaz) [inline patient-roster.md]: What are we planning to do with this payer information?

Comment (97jaz) [reply patient-roster.md]: Also, no care coordinator info?

Comment (santry) [reply patient-roster.md]: Like the PCP info and the care coordinator info (thanks for the reminder), the payer information may be displayed in the future.

**Comment by santry** [comment]
@97jaz updated

**Comment by santry** [comment]
For the identifier type, did you have something in mind? We could formalize the example list into a specific list of allowed values, but I'm not sure what all of those would be …

Unfortunately i'm not sure how quickly we're going to be able to resolve Mt. Sinai, and UNC is waiting on this.

**Comment by 97jaz** [comment]
@santry We could direct them to http://hl7-definition.caristix.com:9010/Default.aspx?version=HL7%20v2.5.1&table=0203

**Comment by 97jaz** [comment]
But that table contains a bunch of stuff we wouldn't really want to see, like, say, someone's Amex number.

**Comment by santry** [comment]
Agreed that we wouldn't want amex number, but we could steer them away from that type. What about the more recent definition here?

https://www.hl7.org/fhir/valueset-identifier-type.html

**Comment by 97jaz** [comment]
@santry Sure, that makes sense

**Pull request merged**